src/rules/__tests__/prefer-importing-jest-globals.test.ts

```diff
@@ -1,16 +1,10 @@
 import dedent from 'dedent';
 import rule from '../prefer-importing-jest-globals';
-import { FlatCompatRuleTester as RuleTester, espreeParser } from './test-utils';
+import { FlatCompatRuleTester as RuleTester } from './test-utils';
 
-const ruleTester = new RuleTester({
```

Comment:
I'm pretty sure when I was working on this all of these tests were passing regardless of this change - overall, we should be retaining the existing test structure, and just adding a new typescript parser based one.

This is a good example of what I mean:

https://github.com/jest-community/eslint-plugin-jest/blob/f969f92a0a3b72df5f87d898d9e3249474fcf8e5/src/rules/__tests__/prefer-to-contain.test.ts#L214-L228

Reply:
Ah, right, I get it now I think!